<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_credit_sum">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>10</y>
     <width>119</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_credit_sum">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>15</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="specialValueText">
    <string notr="true"/>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000044885712678075916785549312.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Anui">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>109</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Dif">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>89</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>129</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Срок (в месяцах)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_period">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>129</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpay">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>264</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_pay">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>300</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percents">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_percents">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>169</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="first_pay">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="overpay">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>264</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="total_pay">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month_pay">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_pay">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
